options/locale/locale_en-US.ini

```diff
@@ -3540,6 +3541,10 @@ workflow.disable_success = Workflow '%s' disabled successfully.
 workflow.enable = Enable Workflow
 workflow.enable_success = Workflow '%s' enabled successfully.
 workflow.disabled = Workflow is disabled.
+workflow.run = Run Workflow
+workflow.not_found = Workflow '%s' not found.
+workflow.run_success = Workflow '%s' run successfully.
+workflow.from_ref = Use workflow from
```

Comment:
Target Ref or other description? `Use workflow from` is incorrect after this change: https://github.com/go-gitea/gitea/pull/28163#issuecomment-1840041721